src/script/blockvalidator.cpp

```diff
@@ -4,6 +4,106 @@
 
 #include "blockvalidator.h"
 
```

Comment:
If we're not going to try to move the code to the same files, can you please at least use the consensus directory instead of the script one?